### DISCUSSION TITLE
Benito van der Zander discussion

### DISCUSSION THREAD

**Discussion by gcarreno** [ObjectPascal-Community/1brc-ObjectPascal#168] [General]
Hey Benito(@benibela),

Here's your initial value on a not so quiet system:
```console
$ ./run_all.sh bzander
******** Run ********

===== Benito van der Zander ======
-- SSD --
Benchmark 1: bzander
  Time (mean ± σ):      8.043 s ±  0.285 s    [User: 193.960 s, System: 10.342 s]
  Range (min … max):    7.421 s …  8.428 s    10 runs
```

[bzander-1_000_000_000-SSD.json](https://github.com/gcarreno/1brc-ObjectPascal/files/15281863/bzander-1_000_000_000-SSD.json)

**Comment by benibela** [comment]
what is "not so quiet"? 

that looks like it is 5-times faster than my laptop

**Reply by gcarreno** [reply]
Hey Benito(@benibela),

When I run quick test I do it with all the apps running on my machine:
- The GNOME Desktop
- The browser with about 5 tabs
- 3 major chat app
- A cryptocoin node because I'm programming for it
- Some code editor, usually either VSCode or Sublime Text

This will introduce some time fluctuations.
But when I run the weekly automation, I log out of the GNOME Desktop and I run it on `tty6`.
This way, the system is as _quiet_ as I can possibly make it and the values are more reliable.

Cheers,
Gus
